tooling/src/main/java/org/opencds/cqf/tooling/acceleratorkit/Processor.java

```diff
@@ -3757,31 +3757,34 @@ public void writeDataElements(String scope, String scopePath, String context) {
         sb.append(System.lineSeparator());
         sb.append(System.lineSeparator());
 
-        sb.append(String.format("using FHIR version '4.0.1'"));
+        sb.append(String.format(retrieveProfileLibrary().get(0)));
         sb.append(System.lineSeparator());
         sb.append(System.lineSeparator());
         sb.append(String.format("include FHIRHelpers version '4.0.1'"));
```

Comment:
There are no format specifiers. (suggest using format, but if the desire is not to, then don't use format)